src/track/track.cpp

```diff
@@ -931,11 +935,13 @@ Track::ExportMetadataResult Track::exportMetadata(
     // be called after all references to the object have been dropped.
     // But it doesn't hurt much, so let's play it safe ;)
     QMutexLocker lock(&m_qMutex);
+    // Normalize metadata before export to adjust the precision of
+    // floating values, ...
```

Comment:
This comment could be a bit more explicit about the issue that the normalization process solves. Adding some words about how the track metadata would be unnecessarily exported without this normalization would be helpful.

Reply:
Ping. Please add a bit more information here, otherwise LGTM.